build/package/Dockerfile.python-toolset

```diff
@@ -0,0 +1,25 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONIOENCODING=UTF-8 \
+    PIP_NO_CACHE_DIR=off \
+    HOME=/app \
+    PATH="/opt/venv/bin:$PATH"
+
+RUN microdnf install -y python38 tar
+
+WORKDIR /app
+
+USER root
+
+RUN python3 -m venv /opt/venv && \

```

Comment:
before installing the venv we should enable nexus globally in the image, like:
https://github.com/opendevstack/ods-quickstarters/blob/38814dfaa64d93f49930b2ab089f54e806f0862c/common/jenkins-agents/python/docker/Dockerfile.ubi8#L16

or:
https://github.com/opendevstack/ods-quickstarters/blob/38814dfaa64d93f49930b2ab089f54e806f0862c/common/jenkins-agents/python/docker/Dockerfile.rhel7#L33

this way as it is done we enable pypi as fallback which is a secure way to handle any issue with nexus

Reply:
@gerardcl good point!

Though really necessary, maybe we can leave this out of this PR and copy the info you gave into a ticket. We need to generally think about how to best provide this info in the builder images (not just python). What do you think?